### PR TITLE
Dtspb 3482 pp to pp

### PR DESCRIPTION
**Pull request by IswaryaPepakayala**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPB-3306
https://tools.hmcts.net/jira/browse/DTSPB-3309
https://tools.hmcts.net/jira/browse/DTSPB-3482

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccdImports/conversionScripts/createAllXLS-pipeline.sh [hunk 1]
```diff
@@ -23,8 +23,9 @@ fi
 echo excludedFilenamePatterns = $excludedFilenamePatterns
 
 export CCD_DEF_CASE_SERVICE_BASE_URL=$1
+export CCD_DEF_AAC_URL=probate-back-office-pr-2347-aac-manage-case-assignment
```
Comment (FeliTam) [inline ccdImports/conversionScripts/createAllXLS-pipeline.sh]: Need to change this line when deploy

File: build.gradle [hunk 2]
```diff
@@ -202,7 +202,7 @@ def versions = [
         lombok                          : '1.18.26',
         springDocOpenUi                 : '1.6.6',
         springSecurityVersion           : '5.7.3',
-        probateCommonsVersion           : '1.4.15',
+        probateCommonsVersion           : 'DTSPB-3309-v5-1.4.10',
```
Comment (DelythJustice) [inline build.gradle]: Just create the newer version and then use - we've done this successfully several times now

File: Jenkinsfile_CNP [hunk 3]
```diff
@@ -61,7 +61,8 @@ def secrets = [
                 secret('levDeathUrl', 'LEV_DEATH_URL'),
                 secret('definition-importer-username', 'CCD_CONFIGURER_IMPORTER_USERNAME'),
                 secret('definition-importer-password', 'CCD_CONFIGURER_IMPORTER_PASSWORD'),
-                secret('probate-storage-account-primary-connection-string', 'PROBATE_STORAGE_ACCOUNT_PRIMARY_CONNECTION_STRING')
+                secret('probate-storage-account-primary-connection-string','PROBATE_STORAGE_ACCOUNT_PRIMARY_CONNECTION_STRING'),
+                secret('probate-storage-account-primary-connection-string', 'MANAGE_CASE_S2S_KEY')
```
Comment (FeliTam) [inline Jenkinsfile_CNP]: This line is duplicate with Line#86 secret('aac-manage-case-s2s-secret', 'MANAGE_CASE_S2S_KEY')

File: src/main/java/uk/gov/hmcts/probate/service/PrepareNocService.java [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+        log.info("Change of Representatives before for case {} : {} ", caseDetails.getId().toString(), representatives);
+        ChangeOfRepresentative representative = buildChangeOfRepresentative(caseData, changeOrganisationRequest);
+        representatives.add(new CollectionMember<>(null, representative));
+        log.info("Change of Representatives after for case {} : {} ", caseDetails.getId().toString(), representatives);
+        representatives.sort((m1, m2) -> {
+            LocalDateTime dt1 = m1.getValue().getAddedDateTime();
+            LocalDateTime dt2 = m2.getValue().getAddedDateTime();
+            return dt1.compareTo(dt2);
+        });
+        Collections.reverse(representatives);
```
Comment (DelythJustice) [inline src/main/java/uk/gov/hmcts/probate/service/PrepareNocService.java]: Can just do: representatives.sort((m1, m2) -> {
    LocalDateTime dt1 = m1.getValue().getAddedDateTime();
    LocalDateTime dt2 = m2.getValue().getAddedDateTime();
    return dt2.compareTo(dt1); // Compare in descending order
}); then don't need to reverse

Comment (IswaryaPepakayala) [reply src/main/java/uk/gov/hmcts/probate/service/PrepareNocService.java]: Modified.Thanks

File: Jenkinsfile_CNP (hunk 3 above)
Comment (IswaryaPepakayala) [reply Jenkinsfile_CNP]: Removed this line.Thanks

File: build.gradle (hunk 2 above)
Comment (IswaryaPepakayala) [reply build.gradle]: Pushed o master and Created tag version.

File: src/integrationTest/java/uk/gov/hmcts/probate/controller/CaveatControllerIT.java [hunk 5]
```diff
@@ -153,7 +154,6 @@ void solsCaveatConfirmation_ShouldReturnDataPayload_OkResponseCode() throws Exce
             .andExpect(content().contentType(MediaType.APPLICATION_JSON));
     }
 
-    @Test
```
Comment (FeliTam) [inline src/integrationTest/java/uk/gov/hmcts/probate/controller/CaveatControllerIT.java]: This line shouldn't be removed

Comment (IswaryaPepakayala) [reply src/integrationTest/java/uk/gov/hmcts/probate/controller/CaveatControllerIT.java]: Thanks for pointed.Added @Test

**Review by FeliTam — APPROVED** [review]

**Pull request merged**